boards/BMS/CMakeLists.txt

```diff
@@ -10,14 +10,11 @@ set(ARM_BINARY_IO_INCLUDE_DIRS
         "Inc/Io"
         )
 
-set(X86_COMPATIBLE_IO_SRCS
-        "${CMAKE_CURRENT_SOURCE_DIR}/Src/Io/Io_VoltageSense.c"
-        "${CMAKE_CURRENT_SOURCE_DIR}/Src/Io/Io_CurrentSense.c")
 set(ARM_BINARY_X86_COMPATIBLE_SRCS
         ${ARM_BINARY_APP_SRCS}
         ${X86_COMPATIBLE_IO_SRCS})
 
-list(REMOVE_ITEM ARM_BINARY_IO_SRCS ${X86_COMPATIBLE_IO_SRCS})
+list(REMOVE_ITEM ARM_BINARY_IO_SRCS "")
```

Comment:
Is this line necessary? There is no item passed to be removed from ARM_BINARY_IO_SRCS.

If this is autogenerated disregard, doesn't hurt anything

Reply:
This line is necessary now that Io_Voltage/CurrentSense files call the Io_Adc code directly
So can no longer test Io_Voltage/CurrentSense

Reply:
Right, but per the `REMOVE_ITEM` syntax, this line isn't doing anything is it? is an item contained in `""`?

https://cmake.org/cmake/help/v3.0/command/list.html